src/main/kotlin/hwr/oop/Deck.kt

```diff
@@ -0,0 +1,27 @@
+package hwr.oop
+
+class Deck {
+    private var deck = ArrayList<Card>()
```

Comment:
It is recommended to use a MutableList rather than a ArrayList.
```suggestion
    private var deck = MutableList<Card>()
```